src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -28,23 +31,10 @@
     public String notificationChannel;
 
     /**
-     * Pending, unverified phone number, in +15551234 format (E.164 format, includes country code, no spaces)
-     * and for which a verification code has been requested.
-     */
-    public String pendingPhoneNumber;
-    /**
-     * Locally-formatted pendingPhoneNumber for display.
-     */
-    public String pendingPhoneNumberFormatted;
-
-    /**
-     * Verified phone number for SMS notifications, in +15551234 format (E.164 format, includes country code, no spaces).
+     * Phone number for SMS notifications.
```

Comment:
Old comment about E.164 format is correct and should be reverted. We want a predictable consistent way for clients to interact with this API. And enforcing that format is an important part of that in my view.

Reply:
Reverted in https://github.com/ibi-group/otp-middleware/pull/83/commits/74075ba31332186ab2906e45a2b02c3ff7906448